learning_assistant/utils.py

```diff
@@ -123,3 +123,16 @@ def get_chat_response(prompt_template, message_list, courserun_id):
         chat = 'Completion endpoint is not defined.'
 
     return response_status, chat
+
+
+def user_role_is_staff(role):
```

Comment:
A bit of forward thinking here. I'm wondering is changing the name to has_permission or something might make us more flexible and account for the future roles and permissions changes coming to the platform. tldr; version is roles would grant permissions and functionality is gated by those permissions, not the roles.